README.md

```diff
@@ -122,7 +123,14 @@ pip install apache-airflow[postgres,google]==1.10.12 \
  --constraint "https://raw.githubusercontent.com/apache/airflow/constraints-1.10.12/constraints-3.7.txt"
 ```
 
+## Official source code
+
+There is only one "official" source code repository for Apache Airflow, like all ASF projects: https://downloads.apache.org. Official releases are cryptographically signed by the release manager and placed in the Apache svn next to the file, its checksum and archived by ASF in case of any legal disputes.
+
+Other ways of retrieving source code are just "convenience" methods. For example, tags in GitHub is only a convenience way to mark that those are the git project sources that were used to generate the "official" source package. What is in the tag is not a concern when it comes to releasing the software, but what matters most is what goes into the official package.
```

Comment:
maybe typos for `tags`/`tag` and `is`/`are`?

Reply:
Thanks - fixed :)